package.json

```diff
@@ -206,6 +212,16 @@
                     "group": "nasc",
                     "when": "config.nasc-touchbar.goToDefinition && !enabledGroup"
                 },
+                {
+					"command": "editor.action.formatDocument",
```

Comment:
Is this block duplicated? We have the same command in line 225, or am I wrong?

Reply:
Yeah, I am not quite sure how this `package.json` is structured... I tried this and it worked 🤦🏻‍♂️

Reply:
I hope that this function is added, I really need it!